<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreSettingsDialog</class>
 <widget class="QDialog" name="ScoreSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Ellenpont szabály beállítások</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>22</y>
          <width>211</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="mainRulesCheckBox">
           <property name="text">
            <string>Alapvető szabályok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="onlyConsonanceCheckBox">
           <property name="text">
            <string>Csak konszonáns hangközök</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="beginEndRuleCheckBox">
           <property name="text">
            <string>Elején és végén csak tökéletes konsz.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maxIntervalCheckBox">
           <property name="text">
            <string>Legnagyobb hangköz T8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nextToTheLastCheckBox">
           <property name="text">
            <string>Utolsó előtti hangköz N6/K3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="modeRuleCheckBox">
           <property name="text">
            <string>Hangnem szabály</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="forbiddenSkipsCheckBox">
           <property name="text">
            <string>Tilos ugrások szabály</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Kotta beállítások</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>23</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Hang távolság</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="noteDistanceSpinBox">
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>label</zorder>
       <zorder>noteDistanceSpinBox</zorder>
       <zorder>horizontalSpacer</zorder>
       <zorder>groupBox_2</zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScoreSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScoreSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
